azure-pipelines.yml

```diff
@@ -55,6 +55,8 @@ steps:
       CNAB_INSTALLATION_NAME: '$(installationName)'
       MANIFEST_FILE: $(cnabRiffDir)/cnab/app/kab/manifest.yaml
       LOG_LEVEL: debug
+  - script: kubectl get pods --all-namespaces | ( ! grep -E "knative-serving|istio-system|riff-system" )
```

Comment:
it might be better to look at the namespaces themselves. Can we pair this with a step that verifies the install created the namespaces

Reply:
done!